src/rpc/protos/omega_edit.proto

```diff
@@ -41,11 +41,38 @@ service Editor {
   rpc UnsubscribeToSessionEvents(ObjectId) returns (ObjectId);
   rpc UnsubscribeToViewportEvents(ObjectId) returns (ObjectId);
 
+  rpc GetCount(CountRequest) returns (CountResponse);
+  rpc GetSessionCount(google.protobuf.Empty) returns (SessionCountResponse);
+
   // Event streams
   rpc SubscribeToSessionEvents(ObjectId) returns (stream SessionEvent);
   rpc SubscribeToViewportEvents(ObjectId) returns (stream ViewportEvent);
 }
 
+enum CountKind {
+  UNDEFINED_COUNT_KIND = 0;
+  COUNT_FILE_SIZE = 1;
```

Comment:
Question: is this meant to match up with `omega_session_get_computed_file_size`?

Reply:
Yes.